tests/Helpers/ErrorLoggerTraitTest.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Tests\Helpers;
+
+use EightshiftLibs\Helpers\ErrorLoggerTrait;
+
+class MockErrorLogger {
+    use ErrorLoggerTrait;
+}
+
+test('Test restResponseHandler will return expected result in case of success', function () {
+    $mockLogger = new MockErrorLogger();
+
+    $response = $mockLogger->restResponseHandler(200, 'Some status', 'Some message', ['fake data']);
+
+    $this->assertJson($response, 'Response is not a JSON string.');
+    $this->assertEquals('[{"code":200,"status":"Some status","message":"Some message","data":["fake data"]},200]', $response);
```

Comment:
If the above changes are implemented then it would be something like this:

```suggestion
    $this->assertEquals('[{"success": "true", "code":200,"status":"Some status","message":"Some message","data":["fake data"]},200]', $response);
```